File: iniciante/1008/1008.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var numero = parseInt(lines.shift());
var quant_hr = parseInt(lines.shift());
var valor_hr = parseFloat(lines.shift());

var salario = quant_hr * valor_hr;

console.log(`NUMBER = ${numero}`);
console.log(`SALARY = U$ ${salario.toFixed(2)}`);

File: iniciante/1008/1008.dart
import 'dart:io';

// RESPOSTA DO BEECROWD (SEM NULL-SAFETY)
/*
void main() {
  int numero = int.parse(stdin.readLineSync());
  int quant_hr = int.parse(stdin.readLineSync());
  double valor_hr = double.parse(stdin.readLineSync());

  double salario = quant_hr * valor_hr;

  print("NUMBER = $numero");
  print("SALARY = U\$ ${salario.toStringAsFixed(2)}");
}
*/

// RESPOSTA COM NULL-SAFETY
void main() {
  int numero = int.parse(stdin.readLineSync()!);
  int quant_hr = int.parse(stdin.readLineSync()!);
  double valor_hr = double.parse(stdin.readLineSync()!);

  double salario = quant_hr * valor_hr;

  print("NUMBER = $numero");
  print("SALARY = U\$ ${salario.toStringAsFixed(2)}");
}
